react-stopwatch-component/src/stopwatch.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+class Stopwatch extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { isTicking: false, time: 0 };
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  startWatch() {
+    this.setState({ time: this.state.time + 1 });
+  }
+
+  handleClick() {
```

Comment:
Trying to understand scoping here (and that I needed to use `this.watchId` was TOUGH.